pydantic_ai_slim/pydantic_ai/models/google.py

```diff
@@ -275,6 +290,9 @@ async def _generate_content(
     def _process_response(self, response: GenerateContentResponse) -> ModelResponse:
         if not response.candidates or len(response.candidates) != 1:
             raise UnexpectedModelBehavior('Expected exactly one candidate in Gemini response')  # pragma: no cover
+        finish_reason_key = response.candidates[0].finish_reason or None
```

Comment:
In the original code, we were calling `.value` on `...[0].finish_reason`. Do we need the same here, and if not, why not?